2009-08-09  Yavor Doganov  <yavor@gnu.org>

	* Tools/GNUmakefile (CONFIG_SYSTEM_LIBS):
	* Tools/make_strings/GNUmakefile (CONFIG_SYSTEM_LIBS): Define
	to the empty string to avoid linking the tools against
	external libraries.


--- gnustep-base-1.20.0.orig/Tools/GNUmakefile
+++ gnustep-base-1.20.0/Tools/GNUmakefile
@@ -102,6 +102,10 @@
 xmlparse_OBJC_FILES = xmlparse.m
 HTMLLinker_OBJC_FILES = HTMLLinker.m
 
+# Reset this variable (defined in config.mak) to avoid useless linkage
+# against the libs gnustep-base uses.
+CONFIG_SYSTEM_LIBS :=
+
 DOCUMENT_NAME = autogsdoc
 
 # Documentation rules for autogsdoc
--- gnustep-base-1.20.0.orig/Tools/make_strings/GNUmakefile
+++ gnustep-base-1.20.0/Tools/make_strings/GNUmakefile
@@ -35,6 +35,10 @@
 	StringsEntry.m \
 	StringsFile.m
 
+# Reset this variable (defined in config.mak) to avoid useless linkage
+# against the libs gnustep-base uses.
+CONFIG_SYSTEM_LIBS :=
+
 include GNUmakefile.preamble
 
 include $(GNUSTEP_MAKEFILES)/tool.make
